packages/react/src/primitives/Loader/Loader.tsx

```diff
@@ -11,10 +11,28 @@ export const LINEAR_FILLED = 'linear-filled';
 export const CIRCULAR_EMPTY = 'circular-empty';
 export const CIRCULAR_FILLED = 'circular-filled';
 
+// circumference = 2 * r * PI  (r = 42)
```

Comment:
Is radius always 42? Should we make 42 a constant and use it in the SVG below?

Reply:
That's true because I set the `stroke-width` to 8% for a good look. `radius` + `stroke-width` = 50%. I can make them constants